fdbserver/workloads/ApiCorrectness.actor.cpp

```diff
@@ -480,6 +481,8 @@ struct ApiCorrectnessWorkload : ApiWorkload {
 	// Gets a single range of values using key selectors from the database and memory store and compares them, returning
 	// true if the results were the same
 	ACTOR Future<bool> runGetRangeSelector(VectorRef<KeyValueRef> data, ApiCorrectnessWorkload* self) {
+		state Reverse reverse = Reverse{ deterministicRandom()->coinflip() };
```

Comment:
The right side use of `Reverse` doesn't seem needed for clarity and initializing to a boolean directly appears to work because it is done on line 438 which presumably compiles?